src/main/java/seedu/duke/userinterface/command/CliCommand.java

```diff
@@ -3,17 +3,29 @@
 import seedu.duke.userinterface.Mode;
 
 public abstract class CliCommand {
-    private Mode uiMode;
-    private String commandParams;
+    protected static Mode applicationState;
+    protected String commandParams;
     public static final String COMMAND_WORD = "";
 
     public abstract void execute();
 
     public void setUiMode(Mode currentUiMode) {
```

Comment:
This should probably be renamed to match the field name, no?

Reply:
Okay, will do